# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../isdn4k-utils/no-mkdev.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./scripts/makedev.sh.vanilla	2000-05-11 12:55:18.000000000 +0200
+++ ./scripts/makedev.sh	2005-07-28 11:22:46.000000000 +0200
@@ -5,6 +5,8 @@
 # This script creates all ISDN devices under /dev .
 # Many/most distributions don't include these devices.
 
+exit 0;
+
 if [ "`id | grep uid=0`" = "" ]; then
 	echo "In order to create device inodes, you must run this script as root."
 	exit 1
